detox/src/devices/drivers/android/genycloud/services/GenyRecipesService.js

```diff
@@ -0,0 +1,30 @@
+const Recipe = require('./dto/GenyRecipe');
+
+class GenyRecipesService {
+  constructor(genyCloudExec, logger) {
+    this.genyCloudExec = genyCloudExec;
+    this.logger = logger;
+  }
+
+  async getRecipeByName(recipeName) {
+    const result = await this.genyCloudExec.getRecipe(recipeName);
+    const { recipes } = result;
+    if (!recipes.length) {
+      return null;
+    }
+
+    if (recipes.length > 1) {
+      this.logger.warn({ event: 'GENYCLOUD_RECIPE_LOOKUP'}, `More than one Genymotion-Cloud recipe found for recipe name ${recipeName}`);
```

Comment:
⚠️ I'd strongly advise printing **what** recipes have been found, otherwise this log is just unhelpful! ⚠️

Reply:
Will do!

Reply:
Done in #2575